docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md

```diff
@@ -80,3 +80,48 @@ on_worker_boot do
   )
 end
 ```
+
+## Initialization Options
+
+The SDK exposes various initialization options which can be set when registering the DevCycle Client:
+
+```ruby
+# Load the gem
+require 'devcycle-ruby-server-sdk'
+
+# Setup authorization
+options = DevCycle::Options.new(enable_edge_db: true, enable_cloud_bucketing: true)
+
+devcycle_client = DevCycle::Client.new("dvc_server_token_hash", options, true)
```

Comment:
everywhere else you use an env pull - not hardcoding

Reply:
this isn't my change, we can do a follow-up change if you'd like to see this differently

Reply:
The diff is attributing it to you; so not sure where else you're seeing this